modules/contact-form/admin.php

```diff
@@ -17,13 +17,13 @@ function grunion_menu_alter() {
 function grunion_media_button( ) {
 	global $post_ID, $temp_ID;
 	$iframe_post_id = (int) (0 == $post_ID ? $temp_ID : $post_ID);
-	$title = esc_attr( __( 'Add a custom form', 'jetpack' ) );
+	$title = esc_html__( 'Add Contact Form', 'jetpack' );
 	$plugin_url = esc_url( GRUNION_PLUGIN_URL );
 	$site_url = esc_url( admin_url( "/admin-ajax.php?post_id={$iframe_post_id}&action=grunion_form_builder&TB_iframe=true&width=768" ) );
 	?>
 
-	<a id="insert-jetpack-contact-form" class="button thickbox" title="<?php esc_html_e( 'Add Contact Form', 'jetpack' ); ?>" data-editor="content" href="<?php echo $site_url ?>&id=add_form">
-		<span class="jetpack-contact-form-icon"></span> <?php esc_html_e( 'Add Contact Form', 'jetpack' ); ?>
+	<a id="insert-jetpack-contact-form" class="button thickbox" title="<?php esc_html_e( $title ); ?>" data-editor="content" href="<?php echo $site_url ?>&id=add_form">
+		<span class="jetpack-contact-form-icon"></span> <?php esc_html_e( $title ); ?>
 	</a>
```

Comment:
@jeherve -- is there a reason I'm missing that we need to double-escape $title?

I would propose that you leave lines 25 & 26 as is, but change 20 to:
`$title = __( 'Add Contact Form', 'jetpack' );`

Thoughts?